molecule/nginx_hardening/prepare.yml

```diff
@@ -2,6 +2,8 @@
 - name: prepare playbook for kitchen testing "ansible-nginx-hardening" with custom settings
   become: true
   hosts: all
+  collections:
```

Comment:
Why is this needed?

Reply:
this was needed on some places, because we refference vars from our roles or include roles. Itseems I put this in a few to many places :D